Dockerfile

```diff
@@ -19,12 +19,13 @@ RUN apt-get update \
 
 RUN pip install \
   pipenv
-  Pillow
 
 WORKDIR /usr/src/app
 
 COPY Pipfile ./
 COPY Pipfile.lock ./
+COPY start_server.sh ./
```

Comment:
I'd put these under all pip install / uninstall cmds in case you modify them then you'll invalidate the docker image layer cache and have to re-install all your deps again, even if they haven't changed.

Reply:
Done!

Reply:
Hm. Not done. In the original order the container builds successfully, but when I move these lines below the pipenv install line I see:

```
[pipenv.exceptions.PipenvOptionsError]:   File "/usr/local/lib/python3.7/site-packages/pipenv/vendor/click/decorators.py", line 17, in new_func
[pipenv.exceptions.PipenvOptionsError]:       return f(get_current_context(), *args, **kwargs)
[pipenv.exceptions.PipenvOptionsError]:   File "/usr/local/lib/python3.7/site-packages/pipenv/cli/command.py", line 254, in install
[pipenv.exceptions.PipenvOptionsError]:       editable_packages=state.installstate.editables,
[pipenv.exceptions.PipenvOptionsError]:   File "/usr/local/lib/python3.7/site-packages/pipenv/core.py", line 1741, in do_install
[pipenv.exceptions.PipenvOptionsError]:       pypi_mirror=pypi_mirror,
[pipenv.exceptions.PipenvOptionsError]:   File "/usr/local/lib/python3.7/site-packages/pipenv/core.py", line 609, in ensure_project
[pipenv.exceptions.PipenvOptionsError]:       validate=validate, skip_requirements=skip_requirements, system=system
[pipenv.exceptions.PipenvOptionsError]:   File "/usr/local/lib/python3.7/site-packages/pipenv/core.py", line 282, in ensure_pipfile
[pipenv.exceptions.PipenvOptionsError]:       "--system is intended to be used for pre-existing Pipfile
ERROR:: --system is intended to be used for pre-existing Pipfile installation, not installation of specific packages. Aborting.
```

Reply:
that certainly seems like a pip error and not related to moving those scripts around. 
Looks like a lockfile is missing?
https://github.com/pypa/pipenv/issues/2183#issuecomment-388567105